frontend/orval.config.ts

```diff
@@ -14,6 +16,7 @@ module.exports = {
             schemas: 'models',
             client: 'swr',
             clean: true,
+            propertySortOrder: 'Alphabetical',
```

Comment:
generates smaller diff vs previous generation. It would be nice to swap that to default `Specification` value, but not in this PR